pyanaconda/payload/dnfpayload.py

```diff
@@ -453,23 +453,37 @@ def addDisabledRepo(self, ksrepo):
         self._add_repo(ksrepo)
         super().addDisabledRepo(ksrepo)
 
-    def _enable_modules(self):
-        """Enable modules (if any)."""
+    def _handle_modules(self):
```

Comment:
I don't like handle, could we switch this to something more descriptive?

Reply:
At the moment (pre-payload modularization) it processes the kickstart module command, so let's rename it that way.